model/task/db.go

```diff
@@ -582,11 +582,10 @@ func GetRecentTasks(period time.Duration) ([]Task, error) {
 		},
 	).Project(
 		bson.M{
-			IdKey:          0,
-			StatusKey:      1,
-			DetailsKey:     1,
-			ActivatedKey:   1,
-			TestResultsKey: 1,
```

Comment:
if the results aren't there then I think this is fine to leave as is. 

Do we want to merge in the new tasks here? or add the lookup?

Reply:
`GetRecentTasks` is called by `GetResultCounts`, which doesn't use the field.